haxepunk/graphics/BitmapText.hx

```diff
@@ -254,14 +227,9 @@ class BitmapText extends Graphic
 	function parseText():Void
 	{
 		// clear current opcode list
-		while (opCodes.length > 0)
-		{
-			opCodes.pop();
-		}
-		while (_scaleStack.length > 0)
-		{
-			_scaleStack.pop();
-		}
+		HXP.clear(opCodes);
```

Comment:
We should definitely update HXP.clear if it isn't "clearing" the array and creating a new one. My reason for changing this code was to reduce duplicate code for clearing arrays.